### PR TITLE
Rename index.md to README.md

### PR DESCRIPTION
**Pull request by SooluThomas**
Renaming index.md file in docs to README.md. Also I think the best place for this is the README file in the root directory. Comments or suggestions appreciated.

### CODE REVIEW

**Comment by JuLieAlgebra** [comment]
@SooluThomas Okie doke, pushed some changes! Let me know what you think. 

I updated the REAMDE link to the Setup pages on the hosted site, moved instructions for building the docs locally to setup.md (since the intended experience is for people to view the documentation online and build docs locally if they're making changes to them and want to test that out before merging?), and modified the mac OS pixi installation instructions based on test setup run with Matt.

Totally fine to add the local-doc-building instructions to the README if you prefer that over the changes I made; give a shout either way :)

**Comment by SooluThomas** [comment]
This looks great. Thank you so much for updating them. We can merge #5 whenever you approve it and then merge this after <3

**Comment by JuLieAlgebra** [comment]
> This looks great. Thank you so much for updating them. We can merge #5 whenever you approve it and then merge this after ❤️

Thanks Soolu! That sounds like a plan :)

**Review by JuLieAlgebra — APPROVED** [review]

**Review by mthanos7 — APPROVED** [review]

**Pull request merged**
